libminifi/test/unit/ExpectedTest.cpp

```diff
@@ -52,7 +52,7 @@ TEST_CASE("expected map", "[expected][map]") {
 
   {
     const nonstd::expected<int, int> e = 21;
-    auto ret = std::move(e) | utils::map(mul2);
+    auto ret = e | utils::map(mul2);
```

Comment:
These are meant to test utilities on rvalues. You can see the same tests above on lvalues. Please revert these changes, except for the last one.

Reply:
Good catch, updated in 7d86623adbe9d1cf018a62e72536bce62cd4332d